theme/ckeditor5-ui/components/tooltip/tooltip.css

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+@import "../../../mixins/_rounded.css";
+
+:root {
+	--ck-tooltip-arrow-size: 5px;
+}
+
+.ck-tooltip {
+	left: 50%;
+
+	/**
+	 * A class once applied displays the tooltip south of the element.
+	 *
+	 *       [element]
+	 *           ^
+	 *     +-----------+
+	 *     |  Tooltip  |
+	 *     +-----------+
+	 */
+	&.ck-tooltip_s {
+		bottom: calc(-1 * var(--ck-tooltip-arrow-size));
+		transform: translateY( 100% );
```

Comment:
Let's speed up that transform: with `translate3d( 0, 100%, 0);`

Reply:
I wouldn't be so sure about it. It needs to be checked because I experienced some issues in the past developing tooltips.

Reply:
Thinking again and you are right this is about tooltip "triangle" and maybe it isn't worth using hardware acceleration (GPU) caused by `will-change/translate3d` property.

Anyway, we need to remember that sometimes we will need to use `will-change/translate3d` - [see this issue with firefox blurred icons](https://github.com/ckeditor/ckeditor5/issues/668).